module/VuFind/src/VuFind/Mailer/Mailer.php

```diff
@@ -333,6 +334,9 @@ public function send($to, $from, $subject, $body, $cc = null, $replyTo = null)
                 $message->addReplyTo($replyTo);
             }
             $this->getTransport()->send($message);
+            if ($this->messageLog) {
+                file_put_contents($this->messageLog, $message->toString() . "\n", FILE_APPEND);
```

Comment:
Possibly add a prefix of some sort as well, to separate the messages.  But I get this is a temp solution and so likely not used beyond testing a message or two.

Reply:
Thanks, @maccabeelevine, that may indeed be helpful. I'm going to merge this as-is for now, though; once I start building tests using this mechanism, it may become obvious whether we need a prefix and how to structure it (e.g. should it be hard-coded, added via a config setting, etc.?). I'll see where the use cases lead me...